paasta_tools/deployd/watchers.py

```diff
@@ -61,27 +61,31 @@ def watch_folder(self, path):
             for child in children:
                 self.watch_folder("{}/{}".format(path, child))
 
+    def _make_service_instance(self, path):
```

Comment:
Can you please call this `_enqueue_service_instance`

Reply:
Sure, will do

Reply:
I saw the watcher being built without this patch, and it called process_node_event. But since event=None, it doesn't pick up when autoscaling is enabled first time. For instances been changed case, autoscaler works fine even without this patch.

Reply:
Gotcha, I think this makes a lot of sense, it is for sure a bug!

I wonder if there's any other bugs that have a similar effect. Let's hope not!

Thanks again for taking a look :-)